<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true">Utils</string>
  </property>
  <widget class="Backdrop" name="backdrop">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>801</width>
     <height>481</height>
    </rect>
   </property>
  </widget>
  <widget class="HeaderLabel" name="headerlabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>241</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Interface Options</string>
   </property>
  </widget>
  <widget class="Label" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>231</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Interface Position</string>
   </property>
  </widget>
  <widget class="Label" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>150</y>
     <width>231</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Interface Orientation</string>
   </property>
  </widget>
  <widget class="Label" name="label_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>250</y>
     <width>231</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Ask Before Discarding
Unsaved Video in RAM</string>
   </property>
  </widget>
  <widget class="Label" name="label_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>330</y>
     <width>441</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
			font-size: 14px;
			background: transparent;
		</string>
   </property>
   <property name="text">
    <string>Pop up a confirmation prompt if you're going to record over a previously-recorded video you haven't viewed in &quot;Play and Save&quot;.</string>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="Label" name="label_5">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>400</y>
     <width>231</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>System Clock</string>
   </property>
  </widget>
  <widget class="Label" name="label_6">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>70</y>
     <width>141</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Example</string>
   </property>
  </widget>
  <widget class="HeaderLabel" name="headerlabel_2">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>20</y>
     <width>211</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Other Options</string>
   </property>
  </widget>
  <widget class="Button" name="uiFileSaving">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>260</y>
     <width>211</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Video Saving</string>
   </property>
   <property name="clickMarginTop" stdset="0">
    <enum>Button::none</enum>
   </property>
   <property name="clickMarginBottom" stdset="0">
    <enum>Button::none</enum>
   </property>
  </widget>
  <widget class="Button" name="uiUserSettings">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>300</y>
     <width>211</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>User Settings</string>
   </property>
   <property name="clickMarginTop" stdset="0">
    <enum>Button::none</enum>
   </property>
   <property name="clickMarginBottom" stdset="0">
    <enum>Button::full</enum>
   </property>
  </widget>
  <widget class="Button" name="uiAboutCamera">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>40</y>
     <width>211</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>About Camera</string>
   </property>
   <property name="clickMarginBottom" stdset="0">
    <enum>Button::none</enum>
   </property>
  </widget>
  <widget class="Button" name="uiUpdateCamera">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>220</y>
     <width>211</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Update Camera</string>
   </property>
   <property name="clickMarginTop" stdset="0">
    <enum>Button::none</enum>
   </property>
   <property name="clickMarginBottom" stdset="0">
    <enum>Button::none</enum>
   </property>
  </widget>
  <widget class="Button" name="uiFactoryUtilities">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>180</y>
     <width>211</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Factory Utilities</string>
   </property>
   <property name="clickMarginTop" stdset="0">
    <enum>Button::none</enum>
   </property>
   <property name="clickMarginBottom" stdset="0">
    <enum>Button::none</enum>
   </property>
  </widget>
  <widget class="Button" name="uiDone">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>400</y>
     <width>181</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string>Done</string>
   </property>
  </widget>
  <widget class="LineEdit" name="uiSystemTime">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>400</y>
     <width>321</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string>2018-09-20 02:53:41 PM</string>
   </property>
  </widget>
  <widget class="ComboBox" name="uiAskBeforeDiscarding">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>270</y>
     <width>271</width>
     <height>81</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Always</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>If Not Reviewed</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Never</string>
    </property>
   </item>
  </widget>
  <widget class="ComboBox" name="uiInterfaceRotated">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>150</y>
     <width>271</width>
     <height>81</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Not Rotated</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Rotated 180Â°</string>
    </property>
   </item>
  </widget>
  <widget class="ComboBox" name="uiInterfaceSide">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>70</y>
     <width>271</width>
     <height>81</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Right-Handed</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Left-Handed</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="uiLayoutPreview">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>90</y>
     <width>191</width>
     <height>111</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../../assets/images/right-handed-interface.svg</pixmap>
   </property>
   <property name="scaledContents">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="Button" name="uiRemoteAccess">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>100</y>
     <width>211</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>App &amp;&amp; Internet</string>
   </property>
   <property name="clickMarginTop" stdset="0">
    <enum>Button::none</enum>
   </property>
   <property name="clickMarginBottom" stdset="0">
    <enum>Button::none</enum>
   </property>
  </widget>
  <widget class="FeedbackLabel" name="uiSystemClockFeedback">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>460</y>
     <width>531</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Date not formatted correctly; format is YYYY-MM-DD HH:MM:SS AM or PM.</string>
   </property>
  </widget>
  <widget class="Button" name="uiStorage">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>140</y>
     <width>211</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Configure Storage</string>
   </property>
   <property name="clickMarginTop" stdset="0">
    <enum>Button::none</enum>
   </property>
   <property name="clickMarginBottom" stdset="0">
    <enum>Button::none</enum>
   </property>
  </widget>
  <zorder>backdrop</zorder>
  <zorder>headerlabel</zorder>
  <zorder>label</zorder>
  <zorder>label_2</zorder>
  <zorder>label_3</zorder>
  <zorder>label_4</zorder>
  <zorder>label_5</zorder>
  <zorder>label_6</zorder>
  <zorder>headerlabel_2</zorder>
  <zorder>uiFileSaving</zorder>
  <zorder>uiUserSettings</zorder>
  <zorder>uiAboutCamera</zorder>
  <zorder>uiUpdateCamera</zorder>
  <zorder>uiFactoryUtilities</zorder>
  <zorder>uiDone</zorder>
  <zorder>uiAskBeforeDiscarding</zorder>
  <zorder>uiInterfaceRotated</zorder>
  <zorder>uiInterfaceSide</zorder>
  <zorder>uiLayoutPreview</zorder>
  <zorder>uiRemoteAccess</zorder>
  <zorder>uiSystemClockFeedback</zorder>
  <zorder>uiSystemTime</zorder>
  <zorder>uiStorage</zorder>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Backdrop</class>
   <extends>QLabel</extends>
   <header>backdrop</header>
  </customwidget>
  <customwidget>
   <class>Button</class>
   <extends>QPushButton</extends>
   <header>button</header>
  </customwidget>
  <customwidget>
   <class>ComboBox</class>
   <extends>QComboBox</extends>
   <header>combo_box</header>
  </customwidget>
  <customwidget>
   <class>FeedbackLabel</class>
   <extends>QLabel</extends>
   <header>feedback_label</header>
  </customwidget>
  <customwidget>
   <class>HeaderLabel</class>
   <extends>QLabel</extends>
   <header>header_label</header>
  </customwidget>
  <customwidget>
   <class>Label</class>
   <extends>QLabel</extends>
   <header>label</header>
  </customwidget>
  <customwidget>
   <class>LineEdit</class>
   <extends>QLineEdit</extends>
   <header>line_edit</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>uiDone</tabstop>
  <tabstop>uiInterfaceSide</tabstop>
  <tabstop>uiInterfaceRotated</tabstop>
  <tabstop>uiAskBeforeDiscarding</tabstop>
  <tabstop>uiSystemTime</tabstop>
  <tabstop>uiAboutCamera</tabstop>
  <tabstop>uiRemoteAccess</tabstop>
  <tabstop>uiStorage</tabstop>
  <tabstop>uiFactoryUtilities</tabstop>
  <tabstop>uiUpdateCamera</tabstop>
  <tabstop>uiFileSaving</tabstop>
  <tabstop>uiUserSettings</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
